### DISCUSSION TITLE
[Config Support]: have recordings enabled, but disabled at startup

### DISCUSSION THREAD

**Discussion by wilcomir** [blakeblackshear/frigate#11688] [Config Support] [answered]
### Describe the problem you are having

Hello,
I manage the recordings enabling/disabling via homeassistant.

I would like to be able to have recordings enabled, so that I can enable/disable them via homeassistant, but I would like recordings to be disabled when frigate starts up.
The use case is that I have cameras inside my house, and I want recordings to be disabled when someone is at home. This is managed via an automation that triggers each time the home occupancy changes, but this does not happen when I restart frigate of course. Currently my workaround is to trigger this automation each time the frigate server goes from unavailable to running, but still I feel there should be a way to properly set this up.

### Version

0.14.0-beta2

### Frigate config file

```yaml
not applicable
```

### Relevant log output

```shell
not applicable
```

### Frigate stats

```json
not applicable
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

**Comment by NickM-27** [comment]
You can create an automation that listens for Frigate starting up and disables it

**Reply by wilcomir** [reply]
Thanks Nick, this is what I currently have implemented. I understand I am not just missing the config option, it simply is not there. Should I file a feature request for this? It’s not like it’s anything urgent or particularly important of course.

**Reply by NickM-27** [reply, marked as answer]
There's already a feature request for it

**Reply by mmerickel** [reply]
@NickM-27 I'm struggling to find this feature request - do you have a link to it so I can track it? Thank you! Maybe I'm wrong but I'm here because I can't find detail anywhere else about how to get recording to be disabled at startup and enabled seprately via HA automations. The blurb in https://docs.frigate.video/configuration/record#can-i-have-continuous-recordings-but-only-at-certain-times doesn't really go into any detail. I'm looking for something more reliable because I really do not want recordings to be enabled unless explicitly via an automation and I'm concerned about lag at startup that could cause it to record even briefly when it shouldn't so I want the default to be off, not on.

**Reply by hawkeye217** [reply]
https://github.com/blakeblackshear/frigate/issues/11314

**Reply by mmerickel** [reply]
Thank you @hawkeye217!
